src/app/pages/system/bootenv/bootenv-form/bootenv-form.component.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  ChangeDetectionStrategy, Component,
+} from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { TranslateService } from '@ngx-translate/core';
+import { Observable } from 'rxjs';
+import { helptext_system_bootenv } from 'app/helptext/system/boot-env';
+import { regexValidator } from 'app/pages/common/entity/entity-form/validators/regex-validation';
+import { EntityUtils } from 'app/pages/common/entity/utils';
+import { BootEnvService, WebSocketService } from 'app/services';
+import { IxModalService } from 'app/services/ix-modal.service';
+
+@UntilDestroy()
+@Component({
+  selector: 'app-bootenv-form',
+  templateUrl: './bootenv-form.component.html',
+  styleUrls: ['./bootenv-form.component.scss'],
+  providers: [BootEnvService, TranslateService],
```

Comment:
Providing components here creates unique instance of these services just for this component.
This is rarely needed.
What you can do is either set `@Injectable({ providedIn: 'root' })` for `BootEnvService` or move `BootEnvService` into `system` and provide it in `SystemModule`.